### PR TITLE
[EventDispatcher] Remove ability to configure tags on RegisterListenersPass

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

Follows #41923

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
A test is failing

**Comment by derrabus** [comment]
I've deleted a bit too much. 🙈 Should be fixed now.

File: src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php [hunk 1]
```diff
@@ -26,9 +26,7 @@
 class RegisterListenersPass implements CompilerPassInterface
 {
     private $hotPathEvents = [];
-    private $hotPathTagName = 'container.hot_path';
     private $noPreloadEvents = [];
-    private $noPreloadTagName = 'container.no_preload';
```
Comment (OskarStark) [inline src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php]: Why no private const?

Comment (derrabus) [reply src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php]: https://github.com/symfony/symfony/pull/40468#issuecomment-845791510

Comment (OskarStark) [reply src/Symfony/Component/EventDispatcher/DependencyInjection/RegisterListenersPass.php]: Thanks 👍🏻

**Comment by chalasr** [comment]
rebase needed

**Comment by nicolas-grekas** [comment]
Thank you @derrabus.

**Pull request merged**